### PR TITLE
Add implied daily growth rate

### LINKED ISSUE #90: Add implied percentage daily growth rate
In the description text at the top. In addition to having implied R0 computed and displayed, also display the implied daily percentage growth rate for both initial and mitigation cases.

### PR DESCRIPTION
**Pull request by ecsmyth**
- Adds `daily_growth` and `daily_growth_t` instance attributes on `Parameters` class.
- Adds `daily_growth_helper` function to `src/penn_chime/parameters.py` to calculate daily growth rate as a function of doubling time.
- Updates `display_header` function in `src/penn_chime/presentation.py` to display initial and mitigated daily growth rates.
- Adds `test_daily_growth` test to `tests/test_app.py` to test above changes

Fixes #90

### CODE REVIEW

**Comment by ecsmyth** [comment]
Set to WIP. I neglected to add tests for the `daily_growth_helper`. I'll get to that tonight.

**Comment by ecsmyth** [comment]
Added tests for `daily_growth_helper` and re-ran commit checks due to Cypress intermittent failure (issue #214)

**Review by cjbayesian — APPROVED** [review]
Awesome. I checked it out and tested locally. Looks perfect.

File: src/penn_chime/parameters.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 `change_date`, so users can see when results have last changed
 """
 
-from numpy import log2  # type: ignore
+from numpy import log2, power  # type: ignore
```
Comment (sam-writer) [inline src/penn_chime/parameters.py]: nit: I think we should `import numpy as np`

Totally nonblocking though

**Review by sam-writer — APPROVED** [review]
LGTM

**Comment by mdbecker** [comment]
@ecsmyth I'm pulling this out of the 1.0 release so that you can resolve the merge conflict. Thanks!

**Review by mdbecker — CHANGES_REQUESTED** [review]
Please resolve merge conflicts and we'll pull this into the next release

File: src/penn_chime/parameters.py (hunk 1 above)
Comment (ecsmyth) [reply src/penn_chime/parameters.py]: Addressed by PR 208

**Review by ecsmyth — COMMENTED** [review]
The current commit addresses the merge conflict. I pushed again after rebasing on develop post PR 208.

**Comment by quinn-dougherty** [comment]
Becker's requested change is just resolving merge conflicts, which are fine, so we're gonna merge in

**Review by quinn-dougherty — APPROVED** [review]

**Pull request merged**